src/test/java/org/alfresco/repo/tagging/TaggingServiceImplTest.java

```diff
@@ -2426,5 +2435,89 @@ public Void execute() throws Throwable
             nodeRefPropInterceptor.setFilterOnGet(true);
         }
     }
+    
+    /**
+     * ALF-21875
+     */
+    @Test
+    public void testTagScopeALF_21875() {
+        
+        TagScopePropertyMethodInterceptor.setEnabled(Boolean.TRUE);
+        AuthenticationUtil.setFullyAuthenticatedUser(AuthenticationUtil.getAdminUserName());
+        String siteName = GUID.generate();
+
+        this.transactionService.getRetryingTransactionHelper().doInTransaction(new RetryingTransactionCallback<Void>() {
+
+                @SuppressWarnings("unchecked")
+                @Override
+                public Void execute() throws Throwable {
+
+                        // STEP1 CreateSite
+                        SiteInfo siteInfo = createSite(siteName, "doclib", SiteVisibility.PUBLIC);
+
+                        // STEP2 upload a document in documentLibrary
+                        NodeRef documentLibraryOfSite = siteService.getContainer(siteInfo.getShortName(), "doclib");
+
+                        NodeRef containerTagScope = fileFolderService
+                                        .create(documentLibraryOfSite, "containerTagScope" + GUID.generate(), ContentModel.TYPE_FOLDER)
+                                        .getNodeRef();
+                        
+        //STEP4 create tag
+        String tagName = GUID.generate();

```

Comment:
Not sure if the rendering in Github is wrong but that looks not in line and I can not see STEP3